src/commands/createFunction/durableSteps/DurableProjectConfigureStep.ts

```diff
@@ -72,6 +72,19 @@ export class DurableProjectConfigureStep<T extends IFunctionWizardContext> exten
                 hostJson.extensions.durableTask = durableUtils.getDefaultNetheriteTaskConfig();
                 await setLocalAppSetting(context, context.projectPath, ConnectionKey.EventHubs, '', MismatchBehavior.Overwrite);
                 break;
+            case DurableBackend.DTS:
+                hostJson.extensions.durableTask = durableUtils.getDefaultDTSTaskConfig();
+                // Non- .NET projects require a special preview extension bundle to work properly
+                // Todo: Remove once this functionality is out of preview
+                if (context.language !== ProjectLanguage.CSharp && context.language !== ProjectLanguage.FSharp) {
+                    hostJson.extensionBundle = {
```

Comment:
I think that the fact we are changing their host bundle version maybe should be output somewhere. Maybe as an activity child or just the output channel?